pyproject.toml

```diff
@@ -23,15 +23,15 @@ python-multipart = "^0.0.5"
 
 
 [tool.poetry.dev-dependencies]
-black = { version = "20.8b1", allow-prereleases = true }
+black = "^21.12b0"
```

Comment:
This should match the `electionguard-python`. We have learned the hard way though not to use `^`. The version of black changes too often. 

https://github.com/microsoft/electionguard-python/blob/db1287d66c8aabf70a9d338bf6c249578906b3df/pyproject.toml#L43